Instagram/Controllers/FeedViewController.swift

```diff
@@ -8,11 +8,86 @@
 import UIKit
 
 /// Feed screen.
-class FeedViewController: UIViewController {
-
+final class FeedViewController: UIViewController {
+  
+  // MARK: - Private enum.
+  private enum Items {
+      case stories
+      case post
+      case recommendation
+  }
+  
+  // MARK: - Private Constants.
+  private enum Constants {
+    static let storyCellId = "StoriesCell"
```

Comment:
```suggestion
    static let storyCellID = "StoriesCell"
```
линтер выключен значит должен был тебя поправить на ID